### PR TITLE
ticdc: fix sink uri for storage sink

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Optimize the descriptions for configuring sink URI

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [x] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/13654
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/ticdc-sink-to-cloud-storage.md [hunk 1]
```diff
@@ -35,34 +35,58 @@ Info: {"upstream_id":7171388873935111376,"namespace":"default","id":"simple-repl
 
 ## Configure sink URI
```
Comment (qiancai) [inline ticdc/ticdc-sink-to-cloud-storage.md:36]: `ticdc/ticdc-changefeed-config.md` in https://github.com/pingcap/docs-cn/pull/13654/files has been translated in a separate PR: https://github.com/pingcap/docs/pull/13357

File: ticdc/ticdc-sink-to-cloud-storage.md [hunk 2]
```diff
@@ -35,34 +35,58 @@ Info: {"upstream_id":7171388873935111376,"namespace":"default","id":"simple-repl
```
Comment (qiancai) [inline ticdc/ticdc-sink-to-cloud-storage.md:35]: `ticdc/ticdc-changefeed-config.md` in https://github.com/pingcap/docs-cn/pull/13654/files has been translated in a separate PR: https://github.com/pingcap/docs/pull/13357

File: ticdc/ticdc-sink-to-cloud-storage.md [hunk 3]
```diff
@@ -35,34 +35,58 @@ Info: {"upstream_id":7171388873935111376,"namespace":"default","id":"simple-repl
...
-This section describes how to configure storage services in the changefeed URI, including Amazon S3, Azure Blob Storage, and NFS.
-
-### Configure external storage
-
-The URI parameters of Amazon S3, GCS, and Azure Blob Storage in TiCDC are the same as their URI parameters in BR. For details, see [Backup storage URI format](/br/backup-and-restore-storages.md#uri-format-description).
-
-### Configure NFS
-
-The following configuration saves row change events to NFS:
+This section describes how to configure Sink URI for storage services, including Amazon S3, GCS, Azure Blob Storage, and NFS. The Sink URI is used to specify the connection information for TiCDC downstream systems and its format is as follows:
```
Comment (qiancai) [inline ticdc/ticdc-sink-to-cloud-storage.md:38]:
```suggestion
This section describes how to configure Sink URI for storage services, including Amazon S3, GCS, Azure Blob Storage, and NFS. The Sink URI is used to specify the connection information for TiCDC downstream systems. The format of Sink URI is as follows:
```

File: ticdc/ticdc-sink-to-cloud-storage.md [hunk 4]
```diff
@@ -35,34 +35,58 @@ Info: {"upstream_id":7171388873935111376,"namespace":"default","id":"simple-repl
...
-This section describes how to configure storage services in the changefeed URI, including Amazon S3, Azure Blob Storage, and NFS.
-
-### Configure external storage
-
-The URI parameters of Amazon S3, GCS, and Azure Blob Storage in TiCDC are the same as their URI parameters in BR. For details, see [Backup storage URI format](/br/backup-and-restore-storages.md#uri-format-description).
-
-### Configure NFS
-
-The following configuration saves row change events to NFS:
+This section describes how to configure Sink URI for storage services, including Amazon S3, GCS, Azure Blob Storage, and NFS. The Sink URI is used to specify the connection information for TiCDC downstream systems. The format of Sink URI is as follows:
```
Comment (CharlesCheung96) [inline ticdc/ticdc-sink-to-cloud-storage.md:38]:
```suggestion
This section describes how to configure Sink URI for storage services, including Amazon S3, GCS, Azure Blob Storage, and NFS. Sink URI is used to specify the connection information of the TiCDC target system. The format is as follows:
```

ref: https://docs.pingcap.com/tidb/dev/ticdc-sink-to-kafka

File: ticdc/ticdc-sink-to-cloud-storage.md [hunk 5]
```diff
@@ -35,34 +35,58 @@ Info: {"upstream_id":7171388873935111376,"namespace":"default","id":"simple-repl
...
+> **Tip:**
+>
+> The URI parameters of Amazon S3, GCS, and Azure Blob Storage in TiCDC are the same as their URI parameters in BR. For details, see [Backup storage URI format](/br/backup-and-restore-storages.md#uri-format-description).
+
+### Configure sink URI for NFS
+
+The following is an example configuration for NFS:
+
+```shell
+--sink-uri="file:///my-directory/prefix?protocol=canal-json"
```
Comment (hfxsd) [inline ticdc/ticdc-sink-to-cloud-storage.md:88]: /// 三个斜杠吗？

File: ticdc/ticdc-sink-to-kafka.md [hunk 6]
```diff
@@ -60,7 +60,7 @@ The following are descriptions of sink URI parameters and values that can be con
...
 | `auto-create-topic` | Determines whether TiCDC creates the topic automatically when the `topic-name` passed in does not exist in the Kafka cluster (optional, `true` by default). |
-| `enable-tidb-extension` | Optional. `false` by default. When the output protocol is `canal-json`, if the value is `true`, TiCDC sends Resolved events and adds the TiDB extension field to the Kafka message. From v6.1.0, this parameter is also applicable to the `avro` protocol. If the value is `true`, TiCDC adds [three TiDB extension fields](/ticdc/ticdc-avro-protocol.md#tidb-extension-fields) to the Kafka message. |
+| `enable-tidb-extension` | Optional. `false` by default. When the output protocol is `canal-json`, if the value is `true`, TiCDC sends [WATERMARK events](/ticdc/ticdc-canal-json.md#watermark-event) and adds the [TiDB extension field](/ticdc/ticdc-canal-json.md#tidb-extension-field) to Kafka messages. From v6.1.0, this parameter is also applicable to the `avro` protocol. If the value is `true`, TiCDC adds [three TiDB extension fields](/ticdc/ticdc-avro-protocol.md#tidb-extension-fields) to the Kafka message. |
```
Comment (hfxsd) [inline ticdc/ticdc-sink-to-kafka.md:63]: 这部分描述是否需要和上面保持一致？https://github.com/pingcap/docs/pull/13445/files#diff-f65513b64fc6da1c3c841d51830087ecde026da67d91e1b38139e258812bbdfdR52

Comment (hfxsd) [inline ticdc/ticdc-sink-to-kafka.md:63]:
```suggestion
| `enable-tidb-extension` | Optional. `false` by default. When the output protocol is `canal-json`, if the value is `true`, TiCDC sends [WATERMARK events](/ticdc/ticdc-canal-json.md#watermark-event) and adds the [TiDB extension field](/ticdc/ticdc-canal-json.md#tidb-extension-field) to Kafka messages. From v6.1.0, this parameter is also applicable to the `avro` protocol. If the value is `true`, TiCDC adds [three TiDB extension fields](/ticdc/ticdc-avro-protocol.md#tidb-extension-fields) to the Avro messages. |
```

File: ticdc/ticdc-sink-to-cloud-storage.md [hunk 7]
```diff
@@ -35,34 +35,58 @@ Info: {"upstream_id":7171388873935111376,"namespace":"default","id":"simple-repl
...
-Other optional parameters in the URI are as follows:
+For `[query_parameters]` in the URI, the following parameters can be configured:
 
 | Parameter | Description | Default value | Value range |
 | :---------| :---------- | :------------ | :---------- |
 | `worker-count` | Concurrency for saving data changes to cloud storage in the downstream.  | `16` | `[1, 512]` |
 | `flush-interval` | Interval for saving data changes to cloud storage in the downstream.   | `5s` | `[2s, 10m]` |
 | `file-size` | A data change file is stored to cloud storage if the number of bytes exceeds the value of this parameter. | `67108864` | `[1048576, 536870912]` |
 | `protocol` | The protocol format of the messages sent to the downstream.  | N/A |  `canal-json` and `csv` |
+| `enable-tidb-extension` | When `protocol` is set to `canal-json` and `enable-tidb-extension` is set to `true`, TiCDC sends [WATERMARK events](/ticdc/ticdc-canal-json.md#watermark-event) and adds the [TiDB extension field](/ticdc/ticdc-canal-json.md#tidb-extension-field) to canal-json messages. | `false` | `false` and `true` |
```
Comment (hfxsd) [inline ticdc/ticdc-sink-to-cloud-storage.md:52]:
```suggestion
| `enable-tidb-extension` | When `protocol` is set to `canal-json` and `enable-tidb-extension` is set to `true`, TiCDC sends [WATERMARK events](/ticdc/ticdc-canal-json.md#watermark-event) and adds the [TiDB extension field](/ticdc/ticdc-canal-json.md#tidb-extension-field) to Canal-JSON messages. | `false` | `false` and `true` |
```

File: ticdc/ticdc-sink-to-cloud-storage.md (hunk 5 above)
Comment (hfxsd) [inline ticdc/ticdc-sink-to-cloud-storage.md:88]: 应该是2个斜杠吧 
```suggestion
--sink-uri="file://my-directory/prefix?protocol=canal-json"
```

Comment (CharlesCheung96) [reply ticdc/ticdc-sink-to-cloud-storage.md:88]: 前两个是协议分隔符，第三个是 linux 根目录

**Review by hfxsd — APPROVED** [review]

**Pull request merged**